### PR TITLE
Update README to point to r2u

### LINKED ISSUE #85: README Update
Getting started with the README, I think [c2d4u](https://launchpad.net/~c2d4u.team/+archive/ubuntu/c2d4u4.0+) has been deprecated in favor of [r2u](https://github.com/eddelbuettel/r2u).

I believe this only impacts the [Ubuntu](https://github.com/cran4linux/bspm#ubuntu) portion of the README, where we should update the PPA to the r2u github and link the install procedure from [here](https://github.com/cran4linux/bspm#ubuntu).

### PR DESCRIPTION
**Pull request by Enchufa2**
Closes #85

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,25 +91,12 @@ $ sudo dnf install R-CoprManager
 ### Ubuntu
 
 There are thousands of binary packages available via the
```
Comment (eddelbuettel) [inline README.md:93]: Can we improve that line?  
```suggestion
(Essentially) all of CRAN is available as binary packages via the
```

File: README.md [hunk 2]
```diff
@@ -91,25 +91,12 @@ $ sudo dnf install R-CoprManager
...
-$ sudo Rscript -e 'install.packages("bspm", repos="https://cran.r-project.org")'
-```
-
-Then, to enable it system-wide (alternatively, use your `.Rprofile`):
-
-```bash
-$ echo "bspm::enable()" | sudo tee -a /etc/R/Rprofile.site
+$ . /etc/os-release # to get UBUNTU_CODENAME
+$ URL="https://raw.githubusercontent.com/eddelbuettel/r2u/master/inst/scripts"
+$ curl -s "${URL}/add_cranapt_${UBUNTU_CODENAME}.sh" | sudo bash -s
```
Comment (eddelbuettel) [inline README.md:99]: Nifty three lines.  I like it.

**Review by eddelbuettel — APPROVED** [review]
Looks good to me.  One line suggestion you can add if you agree.

**Comment by Enchufa2** [comment]
Thanks, committed and merging now.

**Pull request merged**